packages/GA4Client/src/commerce-handler.js

```diff
@@ -30,6 +30,11 @@ var ADD_SHIPPING_INFO = 'add_shipping_info',
     ADD_PAYMENT_INFO = 'add_payment_info',
     VIEW_CART = 'view_cart';
 
+var eventMaxLength = 40,
```

Comment:
These values are already defined in `common.js`. I don't think they need to be in this file since you're not using them here.